src/command_modules/azure-cli-natgateway/azure/cli/command_modules/natgateway/custom.py

```diff
@@ -19,7 +19,7 @@ def network_client_factory(cli_ctx, **kwargs):
 
 def create_nat_gateway(cmd, nat_gateway_name, resource_group_name,
                        location=None, public_ip_addresses=None,
-                       public_ip_prefixes=None, idle_timeout=None, no_wait=False):
+                       public_ip_prefixes=None, idle_timeout=None, no_wait=False, zone=None):
```

Comment:
Have you tested this? Also, should this be zone or zone? Where is the help text?